README.md

```diff
@@ -1,30 +1,34 @@
-# Source
+<img src="assets/logo.png" height="320" width="620" >
 
-A shared components library for Guardian Digital. Uses the Guardian's [Source Design System](https://theguardian.design).
+> A component library for the Guardian's [Source Design System](https://theguardian.design).
 
-## Usage
+## Components ⚛️
 
-Check out our [Getting Started guide](https://theguardian.design/2a1e5182b/p/876251).
+See [the Source storybook](https://guardian.github.io/source) for examples of available components.
```

Comment:
Would be nice to make the Storybook not full screen, so we can see the navbar. The only reason we hide the navbar was so we could embed the stories into Zeroheight. However Zeroheight now allows us to embed stories in a way that removes the navbar, so we don't need to hide it manually ourselves.